runtime/doc/extra-keymaps.txt

```diff
@@ -0,0 +1,29 @@
+*nvim-extra-keymaps.txt*    Extra default keymaps
```

Comment:
The file name is `extra-keymaps.txt`, but here it is referred to as `nvim-extra-keymaps.txt`.